pkgs/data/fonts/pretendard/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchzip }:
+
+let
+  version = "1.3.0";
+
+  mkPretendard = { pname, typeface, sha256 }:
+    fetchzip {
+      name = "${pname}-${version}";
+
+      url = "https://github.com/orioncactus/pretendard/releases/download/v${version}/${typeface}-${version}.zip";
+      inherit sha256;
+
+      postFetch = ''
+        mkdir -p $out/share/fonts/opentype/${typeface}
+        unzip -j $downloadedFile "*.otf" -d $out/share/fonts/opentype/${typeface}/
+      '';
+
+      meta = with lib; {
+        homepage = "https://github.com/orioncactus/pretendard";
+        description = "Pretendard, a system-ui alternative font for all cross-platform.";
```

```suggestion
        description = "System-ui alternative font for all cross-platform";
```
See CONTRIBUTING.md.
However this is still not a proper English sentence.

---

Thanks, now `meta.description` is `An alternative font to system-ui for all platforms`